.github/workflows/cd-dashboard-photon.yml

```diff
@@ -0,0 +1,72 @@
+name: CD Dashboard Photon
+
+on:
+  push:
+    branches:
+      - main
+
+env:
+  SERVICE: sidetree-photon
+  REGION: us-central1
+  PROJECT_ID: ${{ secrets.GCP_PROJECT }}
+  GCP_SA_KEY: ${{ secrets.GCP_SA_KEY }}
+  GCP_RUNNER_SA: ${{ secrets.GCP_SA_RUNNER }}
+  SIDETREE_METHOD: 'photon'
+  MONGO_DB_CONNECTION_STRING: ${{ secrets.MONGO_DB_CONNECTION_STRING }}
+  DATABASE_NAME: ${{ secrets.DATABASE_NAME }}
+  MAX_CONCURRENT_DOWNLOADS: ${{ secrets.MAX_CONCURRENT_DOWNLOADS }}
+  BATCH_INTERVAL_IN_SECONDS: ${{ secrets.BATCH_INTERVAL_IN_SECONDS }}
+  OBSERVING_INTERVAL_IN_SECONDS: ${{ secrets.OBSERVING_INTERVAL_IN_SECONDS }}
+  # photon
```

Comment:
are we use that dashboard can still be built to only support elem?

We want to make sure folks are not forced to run all did methods.

Reply:
Yeah it will work without setting these for others that don't rely on them.  Have tested for .env.ropsten and the test one